Fluid/Filters/MiscFilters.cs

```diff
@@ -97,5 +101,165 @@ public static FluidValue EscapeOnce(FluidValue input, FilterArguments arguments,
+                        case 'r':
+                            result
+                                .Append((value.Hour % 12).ToString(context.CultureInfo).PadLeft(2, '0')).Append(":")
+                                .Append(value.Minute.ToString(context.CultureInfo).PadLeft(2, '0')).Append(":")
+                                .Append(value.Second.ToString(context.CultureInfo).PadLeft(2, '0')).Append(" ")
+                                .Append(value.ToString("tt", context.CultureInfo).ToUpper())
+                                ;
+                            break;
+                        case 'R':
+                            result
+                              .Append(value.Hour.ToString(context.CultureInfo).PadLeft(2, '0')).Append(":")
+                              .Append(value.Minute.ToString(context.CultureInfo).PadLeft(2, '0'));
+                            break;
+                        case 's': result.Append(value.ToUnixTimeSeconds()); break;
+                        case 'S': result.Append(value.Second.ToString(context.CultureInfo).PadLeft(2, '0')); break;
+                        case 'T':
+                            result
+                              .Append(value.Hour.ToString(context.CultureInfo).PadLeft(2, '0')).Append(":")
+                              .Append(value.Minute.ToString(context.CultureInfo).PadLeft(2, '0')).Append(":")
+                              .Append(value.Second.ToString(context.CultureInfo).PadLeft(2, '0'))
+                              ;
+                            break;
+                        case 'u': result.Append((int)value.DayOfWeek); break;
+                        case 'U': result.Append(context.CultureInfo.Calendar.GetWeekOfYear(value.DateTime, System.Globalization.CalendarWeekRule.FirstDay, DayOfWeek.Sunday).ToString().PadLeft(2, '0')); break;
+                        case 'v':
+                            result
+                                .Append(value.Day.ToString(context.CultureInfo).PadLeft(2, ' ')).Append("-")
+                                .Append(context.CultureInfo.DateTimeFormat.AbbreviatedMonthNames[value.Month - 1]).Append("-")
+                                .Append(value.Year)
+                                ;
+                            break;
+                        case 'V': result.Append((value.DayOfYear / 7 + 1).ToString(context.CultureInfo).PadLeft(2, '0')); break;
+                        case 'W': result.Append(context.CultureInfo.Calendar.GetWeekOfYear(value.DateTime, System.Globalization.CalendarWeekRule.FirstDay, DayOfWeek.Monday).ToString().PadLeft(2, '0')); break;
+                        case 'y': result.Append(value.ToString("yy", context.CultureInfo)); break;
+                        case 'Y': result.Append(value.Year); break;
+                        case 'z': result.Append(value.ToString("zzz", context.CultureInfo)); break;
+                        case 'Z': goto default; // unsupported
```

Comment:
I think we need to throw `NotSipportedException` or log a warning here, because the user will expect a certain format